doc/DeveloperGuide.md

```diff
@@ -94,6 +94,11 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Every command should be documented.
+6. The user interface should be intuitive for the user.
+7. Should have all the essential features by November.
```

Comment:
The deadline should be a concrete date. An interesting question arises, should a due date is considered as NFR or not. It definitely influences shareholder in some sense...